evmlib/src/utils.rs

```diff
@@ -1,3 +1,13 @@
+// Copyright 2024 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+#![allow(dead_code)]
```

Comment:
Even though this is a lib crate, clippy says that the `evm_network_from_env` is never used. But it is indeed used by everyone that is calling `evm_network_from_env` 🤔